### ISSUE TITLE
As a marketer, I want an Easy Button Design to Drive Home our Marketing Angle

### ISSUE THREAD

**Issue by owocki** [gitcoinco/Marketing#14]
As a marketer, I want an Easy Button Design to Drive Home our Marketing Angle

We are experimenting with the creation of an 'easy button' that we will use to market Gitcoin at ETHDenver.

The basic premise of it is 'tap button, get a coder"

I just purchased https://www.amazon.com/gp/product/B00HZD86NS/ref=oh_aui_detailpage_o01_s00?ie=UTF8&psc=1 to be our 'easy button' for ETHDEnver

I will need a label for this button.  

See also, this presentation which explains this new direction.
https://docs.google.com/presentation/d/1ODmUmRxLb7mJOCGf-FG6vFIq8HeJj_bZJTiSZBueLuU/edit#slide=id.g4c2a062e5a_0_119

**Comment by owocki** [comment]
my easy buttons came today!  yay :)

heres the insert that came with them

![2019-01-10 11 10 17](https://user-images.githubusercontent.com/513929/50987988-601c6880-14c8-11e9-94af-ce8e676aa2e2.jpg)

**Comment by owocki** [comment]
@PixelantDesign can i work with @willsputra on this?  i could also ask Mladen for help on this too

**Comment by willsputra** [comment]
@owocki something like this?
![asset 1 800x](https://user-images.githubusercontent.com/19514207/51845054-ba834900-2351-11e9-836c-bb2026ad6018.png)

**Comment by owocki** [comment]
@willsputra yes i think that is good!  can i have the SVG so i can order it pls?

**Comment by willsputra** [comment]
@owocki would an .eps file do? downloaded the template from the customeasybutton website
https://drive.google.com/open?id=1w5qngJVMaQT2FXCTvurS5lu9PiF0SRpc
